lib/galaxy/datatypes/converters/interval_to_tabix_converter.py

```diff
@@ -26,13 +26,13 @@ def main():
     # Create index.
     if options.preset:
         # Preset type.
-        ctabix.tabix_index(filename=index_fname, preset=options.preset, keep_original=True,
-                           index_filename=out_fname)
+        pysam.tabix_index(filename=index_fname, preset=options.preset, keep_original=True,
+                          index_filename=out_fname)
     else:
         # For interval files; column indices are 0-based.
-        ctabix.tabix_index(filename=index_fname, seq_col=(options.chrom_col - 1),
-                           start_col=(options.start_col - 1), end_col=(options.end_col - 1),
-                           keep_original=True, index_filename=out_fname)
+        pysam.tabix_index(filename=index_fname, seq_col=(options.chrom_col - 1),
+                          start_col=(options.start_col - 1), end_col=(options.end_col - 1),
+                          keep_original=True, index_filename=out_fname)
```

Comment:
It looks like thay `tabix_index` in https://github.com/pysam-developers/pysam/blob/master/pysam/libctabix.pyx still doesn't support `keep_original` and `index_filename` parameters.

Reply:
That's unfortunate, because it looks like it's almost there: https://github.com/pysam-developers/pysam/blob/master/pysam/libchtslib.pxd#L1355

Reply:
I see there are also some other enhancements in @dannon's branch, like detecting gzip compression instead of relying on the file ending, which would would probably need to include as well. Have we ever tried contributing this back ?

Reply:
let's see how this goes: https://github.com/pysam-developers/pysam/pull/537/files

Reply:
Good catch, I guess I shouldn't have assumed both index methods had been updated :/

If you want, I can PR the gzip detection stuff.

Back when I wrote this, it would have required changes to htslib *and* pysam, and I never got around to doing it.

Reply:
> If you want, I can PR the gzip detection stuff.

That would be great. There's also the `keep_original` setting, which I think we need as well.